Probesys/bileto
# imp: Put documents visually in the TinyMCE editor

## Related issue(s)



https://github.com/Probesys/bileto/issues/149

## Changes



- include the documents in the same "block" as the TinyMCE editor so it looks like being part of the editor
- refactor the JS controllers related to the editor

## How to test manually



- start a new document
- check the attachments are in the editor

## Checklist
